az-digital/az_quickstart
# Closes #714 Add metatag field to all content types

This PR add the field_az_metatag field to the Events, News, Page, and Person content types

## Description
The metatag module in D9 requires that a field is created and added to the content type. This PR creates field_az_metatag and updates the four content types.

@trackleft helped sort out file locations, so that metatag module is not a dependency for these content types.

The default meta description token has been updated to [node:field_az_summary] rather than [node:summary] — we may want to add field_az_summary to person eventually since the other three content types have it.

## Related Issue
#714 

## How Has This Been Tested?
Tested locally and in Probo

Steps to test:
1. Add content (Page, News, Event, Person)
2. Verify that you see the metatag dropdown on the right side of the screen
3. Open metatag and verify that you see the token [node:field_az_summary]
4. Go to summary tab on right hand side. Add text in the summary field
5. Inspect the content and verify that the text from summary was included as the metadescription
6. Edit the content again, go back to metatag and enter a custom metadescription instead of the token
7. Save
8. Inspect the content and verify that the custom metadescription shows up  

## Types of changes


## Checklist:
